dgm-builder/src/main/java/com/cloudbees/groovy/cps/tool/Translator.java

```diff
@@ -389,6 +389,7 @@ public JExpression visitMethodInvocation(MethodInvocationTree mt, Void __) {
                                     .arg(n(e));
                             } else if (overloadsResolved.containsKey(overloadResolved)) {
                                 // Private, so delegate to our mangled version.
+                                // TODO add a String parameter to each internal helper method for the expected methodName to pass to CpsCallableInvocation.<init>
```

Comment:
What do you mean by internal helper method here?

Reply:
The mangled `$each__java_util_Iterator__groovy_lang_Closure` or whatever. It could be improved to take a parameter naming the method name under which we expect `methodCall` to be invoking it. Usually just `each`, but might be `$each__java_util_Iterable__groovy_lang_Closure` for the case that one DGM method is delegating to another.